contracts/pallet-ibc/rpc/src/lib.rs

```diff
@@ -132,6 +133,27 @@ pub struct PacketInfo {
 	pub ack: Option<Vec<u8>>,
 }
 
+impl From<RawPacketInfo> for PacketInfo {
+	fn from(info: RawPacketInfo) -> Self {
+		Self {
+			height: info.height.unwrap_or_default(),
```

Comment:
does default on None makes sense? Why not implement `try_from` instead?

Reply:
Yeah I think that's better